### PR TITLE
Add DGN to Terra, Migaloo; Register IBC Channels

### PR DESCRIPTION
**Pull request by calabiyau87**
Add DGN IBC:
Terra:
ibc/B2AA4C3CD19954859C3B537EC0705640AFC01075F52993D9AC5E73F07F0386CC

Migaloo:
ibc/D24B4564BCD51D3D02D9987D92571EAC5915676A9BD6D9B0C1D0254CB8A5EA34

Register IBC Channels:

dungeon channel-6 <> migaloo channel-142
dungeon channel-7 <> terra2 channel-582

### CODE REVIEW

File: migaloo/assetlist.json [hunk 1]
```diff
@@ -865,6 +865,58 @@
...
+          "chain": {
+            "channel_id": "channel-142",
+            "path": "transfer/channel-142/uluna"
+          }
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/dungeon/images/DGN.png"
+      },
+      "coingecko_id": "dragon-coin-2",
```
Comment (JeremyParish69) [inline migaloo/assetlist.json:903]: Please remove the coingecko id here. It's already been registered to the original Dragon coin in Dungeon Chain.

File: migaloo/assetlist.json [hunk 2]
```diff
@@ -865,6 +865,58 @@
...
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "dungeon",
+            "base_denom": "udgn",
+            "channel_id": "channel-6"
+          },
+          "chain": {
+            "channel_id": "channel-142",
```
Comment (JeremyParish69) [inline migaloo/assetlist.json:895]: Please register this IBC connection in the _IBC directory.

File: migaloo/assetlist.json (hunk 1 above)
Comment (calabiyau87) [reply migaloo/assetlist.json:903]: I'm new to the world of Github and PRs. How do I make this change without creating a whole other PR?

Comment (calabiyau87) [reply migaloo/assetlist.json]: I changed this in my patch-1 fork and committed it

File: migaloo/assetlist.json (hunk 2 above)
Comment (JeremyParish69) [reply migaloo/assetlist.json:895]: No, it cannot be added before that registration

**Comment by calabiyau87** [comment]
@JeremyParish69 All necessary information has been added along with the IBC channesl

File: terra2/assetlist.json [hunk 3]
```diff
@@ -1643,4 +1643,57 @@
       }
     }
   ]
-}
\ No newline at end of file
+},
+{
```
Comment (JeremyParish69) [inline terra2/assetlist.json:1647]: This asset needs to go inside the `assets` array

Comment (calabiyau87) [reply terra2/assetlist.json]: @JeremyParish69 Corrected

File: _IBC/dungeon-migaloo.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  "chain_2": {
+    "chain_name": "migaloo",
+    "client_id": "07-tendermint-139",
+    "connection_id": "connection-109"
+  },
+  "channels": [
+    {
+      "chain_1": {
+        "channel_id": "channel-6",
```
Comment (JeremyParish69) [inline _IBC/dungeon-migaloo.json:16]: Error: For chain: dungeon, channel_id: channel-6 is registered for both: migaloo and terra2.

Comment (calabiyau87) [reply _IBC/dungeon-migaloo.json:16]: @JeremyParish69 Corrected to channel-7

File: terra2/assetlist.json [hunk 5]
```diff
@@ -1641,6 +1641,57 @@
...
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "dungeon",
+            "base_denom": "udgn",
+            "channel_id": "channel-7"
+          },
+          "chain": {
+            "channel_id": "channel-582",
+            "path": "transfer/channel-6/udgn"
```
Comment (JeremyParish69) [inline terra2/assetlist.json:1672]: This path is incorrect. I believe it should be transfer/channel-582/udgn

Comment (calabiyau87) [reply terra2/assetlist.json]: @JeremyParish69 Corrected

**Pull request merged**